src/test/java/com/example/project/item/Item.java

```diff
@@ -0,0 +1,15 @@
+package com.example.project.item;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Item {
+    private String name;
+    private double price;
```

Comment:
I would say it is better to use wrapper classes instead of primitive types. As primitive types are instantiated with the default values and that might not be what you expect.